.github/CODEOWNERS

```diff
@@ -1,100 +1,100 @@
+src/sinks/kafka/ @vectordotdev/vector
+src/sinks/logdna.rs @vectordotdev/vector
+src/sinks/loki/ @vectordotdev/vector
+src/sinks/nats.rs @vectordotdev/vector
+src/sinks/new_relic/ @vectordotdev/vector # sink_newrelix,sink_newrelic_logs
+src/sinks/papertrail.rs @vectordotdev/vector
+src/sinks/prometheus/ @vectordotdev/vector # sink_prometheus_exporter,sink_prometheus_remote_write
+src/sinks/pulsar.rs @vectordotdev/vector
+src/sinks/redis.rs @vectordotdev/vector
+src/sinks/sematext/ @vectordotdev/vector # sink_sematext_logs,sink_sematext_metrics
+src/sinks/socket.rs @vectordotdev/vector
+src/sinks/splunk_hec/ @vectordotdev/vector # sink_splunk_hec_logs,sink_splunk_hec_metrics
+src/sinks/statsd.rs @vectordotdev/vector
+src/sinks/vector/ @vectordotdev/vector
+src/sinks/websocket/ @vectordotdev/vector
+src/source_sender/ @vectordotdev/vector
+src/sources/ @vectordotdev/vector
+src/sources/amqp.rs @vectordotdev/vector
+src/sources/apache_metrics/ @vectordotdev/vector
+src/sources/aws_ecs_metrics/ @vectordotdev/vector
+src/sources/aws_kinesis_firehose/ @vectordotdev/vector
+src/sources/aws_s3/ @vectordotdev/vector
+src/sources/aws_sqs/ @vectordotdev/vector
+src/sources/datadog_agent/ @vectordotdev/vector
+src/sources/demo_logs.rs @vectordotdev/vector
+src/sources/dnstap/ @vectordotdev/vector
+src/sources/docker_logs/ @vectordotdev/vector
+src/sources/eventstoredb_metrics/ @vectordotdev/vector
+src/sources/exec/ @vectordotdev/vector
+src/sources/file.rs @vectordotdev/vector
+src/sources/file_descriptors/ @vectordotdev/vector # source_file_descriptor,source_stdin
+src/sources/fluent/ @vectordotdev/vector
+src/sources/gcp_pubsub.rs @vectordotdev/vector
+src/sources/heroku_logs.rs @vectordotdev/vector
+src/sources/host_metrics/ @vectordotdev/vector
+src/sources/http_client/ @vectordotdev/vector
+src/sources/http_server.rs @vectordotdev/vector
+src/sources/internal_logs.rs @vectordotdev/vector
+src/sources/internal_metrics.rs @vectordotdev/vector
+src/sources/journald.rs @vectordotdev/vector
+src/sources/kafka.rs @vectordotdev/vector
+src/sources/kubernetes_logs/ @vectordotdev/vector
+src/sources/logstash.rs @vectordotdev/vector
+src/sources/mongodb_metrics/ @vectordotdev/vector
+src/sources/nats.rs @vectordotdev/vector
+src/sources/nginx_metrics/ @vectordotdev/vector
+src/sources/opentelemetry/ @vectordotdev/vector
+src/sources/postgresql_metrics.rs @vectordotdev/vector
+src/sources/prometheus/ @vectordotdev/vector # source_prometheus_remote_write,source_prometheus_scrape
+src/sources/redis/ @vectordotdev/vector
+src/sources/socket/ @vectordotdev/vector
+src/sources/splunk_hec/ @vectordotdev/vector
+src/sources/statsd/ @vectordotdev/vector
+src/sources/syslog.rs @vectordotdev/vector
+src/sources/vector/ @vectordotdev/vector
+src/test_util/ @vectordotdev/vector
+src/topology/ @vectordotdev/vector
+src/transforms/ @vectordotdev/vector
 website/ @vectordotdev/ux-team @vectordotdev/documentation
```

Comment:
Ditto `documentation`

Reply:
This one is still used since there is a separate set of individuals that reviews documentation changes.